apps/web/package.json

```diff
@@ -55,6 +55,7 @@
     "dotenv": "^16.0.3",
     "framer-motion": "^10.12.9",
     "graphql": "^16.6.0",
+    "idb-keyval": "^6.2.0",
```

Comment:
let's do this migration as a fast-follow; this PR has been open awhile so it would be great to get what we have in before additional merge conflicts come up.